README.md

```diff
@@ -72,19 +72,12 @@ Seeking review of the code that verifies there is no overpayment. Contributions
 
 ## Minimum Supported Rust Version (MSRV)
 
-The `payjoin` library should always compile with any combination of features on Rust **1.57.0**.
-
-To build and test with the MSRV you will need to pin the below dependency versions:
-
-```shell
-cargo update -p log --precise 0.4.18
-cargo update -p tempfile --precise 3.6.0
-cargo update -p flate2 --precise 1.0.26
-cargo update -p minreq --precise 2.8.0
-cargo update -p rustls --precise 0.20.8
-cargo update -p webpki --precise 0.22.2
-cargo update -p byteorder --precise 1.4.3
-```
+The `payjoin` library MSRV is **1.63.0**.
+
+The `payjoin` library is reguraly tested aginst **1.63.0/stable/nightly**.
+
+The `payjoin-cli` package is not maintaing MSRV and 
```

Comment:
in `payjoin-cli` we use rolluier and clap which both maintain MSRV of latest 3 versions(including stable).. so it requires a bit of work to make same msrv as `payjoin`.

Reply:
Good point. I'd like to move `rouille` to `hyper` anyhow because we're using it for `payjoin-relay`, but I'm not sure if it can be done in a blocking fashion without tokio or with a low msrv. We can definitely pin deps to use [an older version of clap for msrv 1.63 as bdk-cli does](https://github.com/bitcoindevkit/bdk-cli/blob/master/Cargo.toml#L17).